pkg/repositories/errors/postgres.go

```diff
@@ -13,7 +13,7 @@ import (
 	"github.com/prometheus/client_golang/prometheus"
 
 	"github.com/flyteorg/flyteadmin/pkg/errors"
-	"github.com/jinzhu/gorm"
+	"gorm.io/gorm"
```

Comment:
can you add the changes from https://github.com/flyteorg/datacatalog/commit/11d64b3479a33a60a77a3232dad2b13489ebef5a#diff-7ea834bee5dcbc7c2b02c416df4ecb4e5d85f4776311e6d114b64140cbcb1251R43 to this file too? 

we ran into issues after the gorm upgrade that necessitated adding them in

Reply:
Done